katello-attach-subscription

```diff
@@ -15,9 +15,12 @@
 # Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston,
 # MA 02110-1301, USA.
 
+require 'rubygems'
+require 'bundler/setup'
 require 'optparse'
 require 'yaml'
 require 'apipie-bindings'
+#require 'pry'
```

Comment:
what are those new requires for? I'd prefer not to have any gems loaded that are not shipped in RHEL or the satellite6-tools repo.